.github/workflows/env-setup-deploy-forms-server.yml

```diff
@@ -26,7 +22,7 @@ jobs:
     env:
       FORMS_NAME: forms
       NAMESPACE: ${{ secrets.OPENSHIFT_ENV_NAMESPACE }}
-      BUILD_REF: ${{ inputs.formioTag }}
+      FORMIO_SOURCE_REPO_TAG: ${{ inputs.formioTag }}
```

Comment:
We echo `BUILD_REF` at line 38. I must be updated to `FORMIO_SOURCE_REPO_TAG`

Reply:
Removed since the value form `inputs.formioTag` is printed even in the `run-name`.